vllm/model_executor/layers/quantization/awq_triton.py

```diff
@@ -0,0 +1,320 @@
+import torch
+import triton
+import triton.language as tl
+
+
+@triton.jit
+def awq_dequantize_kernel(
+        qweight_ptr,  # quantized matrix
+        scales_ptr,  # scales, per group
+        zeros_ptr,  # zeros, per group
+        split_k_iters,  # Not used
+        thx,  # Not used
+        thy,  # Not used
+        group_size,  # Should always be 128
```

Comment:
Maybe remove the comment about "always being 128"...  This should work for any reasonable value.

Reply:
The group size is set when the model is quantized and it's always 128.  I can update the comment though.